game/scripts/vscripts/Sections/Chapter2/SectionOpening.ts

```diff
@@ -22,51 +44,119 @@ const onStart = (complete: () => void) => {
+            tg.goToLocation(Vector(-6574, -3742, 256)),
+            tg.immediate(context => {
+                context[ChapterTwoOpeningGoalKeys.MoveNextToBarracks] = GoalState.Completed,
+                    context[ChapterTwoOpeningGoalKeys.ListenToSunsfanAndSlacks] = GoalState.Started
+            }),
+            tg.immediate(() => playerHero.Stop()),
+            tg.immediate(() => canPlayerIssueOrders = false),
+            tg.textDialog(LocalizationKey.Script_2_Opening_1, context => context[CustomNpcKeys.SlacksMudGolem], 10),
+            tg.textDialog(LocalizationKey.Script_2_Opening_2, context => context[CustomNpcKeys.SunsFanMudGolem], 3),
+            tg.textDialog(LocalizationKey.Script_2_Opening_3, context => context[CustomNpcKeys.SlacksMudGolem], 5),
+            tg.textDialog(LocalizationKey.Script_2_Opening_4, context => context[CustomNpcKeys.SunsFanMudGolem], 8),
+            tg.textDialog(LocalizationKey.Script_2_Opening_5, context => context[CustomNpcKeys.SlacksMudGolem], 6),
+            tg.textDialog(LocalizationKey.Script_2_Opening_6, context => context[CustomNpcKeys.SunsFanMudGolem], 14),
+            tg.textDialog(LocalizationKey.Script_2_Opening_7, context => context[CustomNpcKeys.SlacksMudGolem], 8),
+            tg.textDialog(LocalizationKey.Script_2_Opening_8, context => context[CustomNpcKeys.SunsFanMudGolem], 12),
+            tg.textDialog(LocalizationKey.Script_2_Opening_9, context => context[CustomNpcKeys.SlacksMudGolem], 12),
+            tg.textDialog(LocalizationKey.Script_2_Opening_10, context => context[CustomNpcKeys.SunsFanMudGolem], 18),
+            tg.textDialog(LocalizationKey.Script_2_Opening_11, context => context[CustomNpcKeys.SlacksMudGolem], 5),
+            tg.textDialog(LocalizationKey.Script_2_Opening_12, context => context[CustomNpcKeys.SunsFanMudGolem], 3),
+            tg.textDialog(LocalizationKey.Script_2_Opening_13, context => context[CustomNpcKeys.SlacksMudGolem], 6),
+            tg.fork(_ => radiantCreepsNames.map(unit => tg.spawnUnit(unit, Vector(-6795, -3474, 256), DotaTeam.GOODGUYS, undefined))),
+            tg.fork(_ => direCreepNames.map(unit => tg.spawnUnit(unit, Vector(-5911, 5187, 128), DotaTeam.BADGUYS, undefined))),
+            tg.immediate(_ => {
+                // Group radiant creeps
+                const creeps = Entities.FindAllByClassname("npc_dota_creature") as CDOTA_BaseNPC[];
+
+                for (const creep of creeps) {
+                    if (creep.GetUnitName() == CustomNpcKeys.RadiantMeleeCreep || creep.GetUnitName() == CustomNpcKeys.RadiantRangedCreep) {
+                        radiantCreeps.push(creep);
+                    }
+
+                    if (creep.GetUnitName() == CustomNpcKeys.DireMeleeCreep || creep.GetUnitName() == CustomNpcKeys.DireRangedCreep) {
+                        direCreeps.push(creep);
+                    }
+                }
+                print(radiantCreeps.length)
```

Comment:
Could you remove this, think we found the problem and it's no longer required.

Reply:
Done in latest commit.